docs/_components/Primary Nav.md

```diff
@@ -1,8 +1,9 @@
 ---
 title: Primary Nav
+category: components
 status: early draft
 description: Navigation component w/ primary and secondary variants.
-order: 1
+order: 17
```

Comment:
I am wondering why the ordering of this and the `docs/_elements/Button.md` changed below